src/trackers/FNP.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+# import discord
+import asyncio
+import requests
+import distutils.util
+import os
+import platform
+
+from src.trackers.COMMON import COMMON
+from src.console import console
+
+
+class FNP():
+    """
+    Edit for Tracker:
+        Edit BASE.torrent with announce and source
+        Check for duplicates
+        Set type/category IDs
+        Upload
+    """
+
+    ###############################################################
+    ########                    EDIT ME                    ########
+    ###############################################################
+
+    # ALSO EDIT CLASS NAME ABOVE
+
+    def __init__(self, config):
+        self.config = config
+        self.tracker = 'FNP'
+        self.source_flag = 'FnP'
+        self.upload_url = 'https://fearnopeer.com/api/torrents/upload'
+        self.search_url = 'https://fearnopeer.com/api/torrents/filter'
+        self.signature = 'This torrent was downloaded from FearNoPeer.'
+        self.banned_groups = [""]
+        pass
+    
+    async def get_cat_id(self, category_name):
+        category_id = {
+            'MOVIE': '1', 
+            'TV': '2', 
+            }.get(category_name, '0')
+        return category_id
+
+    async def get_type_id(self, type):
+        type_id = {
+            'DISC': '1', 
+            'REMUX': '2',
+            'WEBDL': '4', 
+            'WEBRIP': '5', 
+            'HDTV': '6',
+            'ENCODE': '3'
+            }.get(type, '0')
+        return type_id
+
+    async def get_res_id(self, resolution):
+        resolution_id = {
+            '8640p':'10', 
```

Comment:
![image](https://github.com/L4GSP1KE/Upload-Assistant/assets/1276544/5a72c4c5-e87b-41b8-b55b-9fedca9170fd)
Only the following video resolutions are supported by this PT, should we map 480i and 480p to the same resolution id, or map it to `10` or `other`?

Will get the following error if 480i is not mapped
```python
No dupes found
{'success': False, 'message': 'Validation Error.', 'data': {'resolution_id': ['The selected resolution id is invalid.']}}
Adding to qbit
Adding and rechecking torrent
```